nav2_bt_navigator/behavior_trees/navigate_w_replanning.xml

```diff
@@ -4,7 +4,7 @@
 
 <root main_tree_to_execute="MainTree">
   <BehaviorTree ID="MainTree">
-    <Sequence name="root">
+    <Sequence name="NavigateSequence">
```

Comment:
No issue with the change, but why?

Reply:
These names are the names that are used in the log. If no name is used, the node type is used. I wanted to prevent any confusion in the log between the actual `<root>` node and this node.